database/buildSchema/44_data_view.sql

```diff
@@ -25,6 +26,7 @@ CREATE TABLE data_view_column_definition (
     element_parameters jsonb,
     additional_formatting jsonb,
     value_expression jsonb,
+    sort_column varchar,
```

Comment:
If the view column is not a "basic" column (i.e. doesn't correspond to a physical column on the data table), then it won't be sortable. This allows us to specify an "actual" column to sort by if you want it to be.

For example: the "addressFull" field in the "organisation" view is a combination of multiple database fields. So I've specified the "address" column as the sort column. Now when you click the header for the address, it'll sort by that field.